src/app.js

```diff
@@ -1,21 +1,64 @@
+/* eslint-disable no-useless-escape */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const PORT = process.env.PORT || 8080;
+
+const mineTipe = {
+  html: 'text/html',
+  css: 'text/css',
+  js: 'text/javascript',
+  xml: 'text/xml',
+  json: 'application/json',
+  pdf: 'application/pdf',
+};
+
+const server = http.createServer((req, res) => {
+  const normalizeUrl = new URL(req.url, `http://${req.headers.host}`);
+  const fileName = normalizeUrl.pathname.slice(1).split('/');
```

Comment:
fileName sound as string, but you set an array here
so better to rename it to `fileNameParts`